<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>串口设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="combo_serialNum"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refreshSerial">
         <property name="text">
          <string>刷新串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_switchSerial">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>操作</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_reqList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>加载变量列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_switchDebug">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>开始调试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_freq">
         <item>
          <property name="text">
           <string>200Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="数据刷新频率">100Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10Hz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="数据刷新频率">1Hz</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="table_var"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
